test/integration/targets/dnf5/playbook.yml

```diff
@@ -0,0 +1,19 @@
+- hosts: localhost
+  tasks:
+    - block:
+        - command: "dnf install -y 'dnf-command(copr)'"
+        - command: dnf copr enable -y rpmsoftwaremanagement/dnf5-unstable
+        - command: dnf install -y python3-libdnf5 python3-libdnf5-cli dnf5
+
+        - include_role:
+            name: dnf
+          vars:
+            dnf5: true
+            dnf_log_files:
+              - /var/log/dnf5.log
+      when:
+        - ansible_distribution == 'Fedora'
+        - ansible_distribution_major_version is version('37', '==')
```

Comment:
This should check for >= 37, otherwise when we drop 37 from CI the tests will be skipped.